Dockerfile

```diff
@@ -2,21 +2,20 @@ ARG base_image=alpine
 ARG builder_image=golang
 
 FROM ${builder_image} as builder
-ADD . /go/src/github.com/aoldershaw/github-pr-resource
 WORKDIR /go/src/github.com/aoldershaw/github-pr-resource
 RUN curl -sL https://taskfile.dev/install.sh | sh
+ADD . /go/src/github.com/aoldershaw/github-pr-resource
```

Comment:
Reordered so we don't have to keep installing the install script every time we change the Go code.